### ISSUE TITLE
Water Physics

### ISSUE THREAD

**Issue by CryptoMorin** [Prunt/RestrictedCreative#31]
You cannot place water on certain blocks.
One of them is glass.

There is no restriction for WATER_BUCKET in the config and when I disabled the plugin this stopped.
This is really annoying and happens randomly with some other blocks as well.
Please fix this ASAP.

**Comment by Prunt** [comment]
I've disabled placing water on some blocks because they can break and drop. These blocks include torches, ladders and such, they are intended. But yes, glass is an issue right now and I'l fix it in the next version.

**Comment by Prunt** [comment]
I have found a fix for glass, but you mentioned that you have problems with some other blocks as well. Can you give me a list of the blocks you can't place water on?

**Comment by CryptoMorin** [comment]
Unfortunately not.
This happens randomly. The only block that my players found that will always do this is only glass.
I'll be asking them to find more.

For examples, stone. I was placing water repeatedly to see if it's a visual glitch or not (caused by lag).
I tried deopping, opping, placing blocks, breaking blocks. It just didn't work and I gave up after an hour.
The thing that made this extremely weird that I completely stopped trying, is that it just happens in small regions. It wasn't even near spawn, the whole place was far away in wilderness.
Tried 5-10 blocks around it, and it worked fine...
